api/src/main/java/com/basistech/rosette/api/RosetteAPI.java

```diff
@@ -1216,4 +1839,26 @@ private String headerValueOrNull(Header header) {
     public void close() throws IOException {
         httpClient.close();
     }
+
+    private DocumentRequest.Builder contentAndOptionsBuilder(InputStream stream, String contentType,
```

Comment:
The java naming convention calls for a verb in method names. So how about just calling it `getDocumentRequestBuilder`?